module/observationSubmissions/helper.js

```diff
@@ -55,14 +55,13 @@ module.exports = class ObservationSubmissionsHelper {
           });
         }
 
-        let submissionDocuments;
-
-        if (sortedData !== 'all') {
+        let submissionDocuments; 
+        if (sortedData !== 'all') { 
           submissionDocuments = await database.models.observationSubmissions
             .find(queryObject, projection)
             .sort(sortedData)
             .lean();
-        } else {
+        } else { 
           submissionDocuments = await database.models.observationSubmissions.find(queryObject, projection).lean();
```

Comment:
@borkarsaish65 we should have a file called observationSubmissionQueries. and should ba calling models from there

Reply:
This is the way it has been already I didnt change or added this new function. I will note this down and make changes later as it will delay my deadline.